src/components/text/text.stories.tsx

```diff
@@ -0,0 +1,91 @@
+import { Meta, StoryFn } from '@storybook/react';
+import Text from './text';
+import {
+	fontWeightClassNames,
+	fontSizeClassNames,
+	lineHeightClassNames,
+	letterSpacingClassNames,
+	fontColorClassNames,
+} from './styles';
+
+export default {
+	title: 'Atoms/Text',
+	component: Text,
+	tags: [ 'autodocs' ],
+	argTypes: {
+		weight: {
+			control: 'select',
+			options: Object.keys( fontWeightClassNames ).map( Number ),
+			description: 'The font weight of the text',
+		},
+		size: {
+			control: 'select',
+			options: Object.keys( fontSizeClassNames ).map( Number ),
+			description: 'The font size of the text',
+		},
+		lineHeight: {
+			control: 'select',
+			options: Object.keys( lineHeightClassNames ).map( Number ),
+			description: 'The line height of the text',
+		},
+		letterSpacing: {
+			control: 'select',
+			options: Object.keys( letterSpacingClassNames ).map( Number ),
+			description: 'The letter spacing of the text',
+		},
+		color: {
+			control: 'select',
+			options: Object.keys( fontColorClassNames ),
+			description: 'The font color of the text',
+		},
+		as: {
+			control: 'select',
+			options: [ 'p', 'span', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'div' ],
```

Comment:
Should be include label here, or this is okay?

Reply:
Yes, need to include the label. These are included only as an example so that the developer knows that they can pass any valid tag or component.